docs/source/setup-guide.rst

```diff
@@ -35,19 +35,19 @@ Step 2: Install Vivado and Renode
 Step 3: Install RISCV toolchain
 -------------------------------
 
-1. Download the `April 2020`_ toolchain from freedom-tools and unpack the binaries to your home directory:
+1. Download the `August 2020`_ toolchain from freedom-tools and unpack the binaries to your home directory:
 
-.. _`April 2020`: https://github.com/sifive/freedom-tools/releases/tag/v2020.04.0-Toolchain.Only
+.. _`April 2020`: https://github.com/sifive/freedom-tools/releases/tag/v2020.08.0
```

Comment:
April -> August

Reply:
Oops, I rushed this PR and it shows. Luckily we have CI to catch this mistake!